src/components/Calendar/Calendar.js

```diff
@@ -1,5 +1,28 @@
-import React from "react";
+import React, { Component } from "react";
+import { Calendar, momentLocalizer } from "react-big-calendar";
+import moment from "moment";
+import events from "./Events";
 
-export default function Calendar() {
-  return <div></div>;
+import "react-big-calendar/lib/css/react-big-calendar.css";
+
+const localizer = momentLocalizer(moment);
+
+export class BetterCalendar extends Component {
```

Comment:
Any specific reason you made this a class component?

Reply:
It was in the sample code that I used. I just changed it to a functional component.

Reply:
Sweet, yeah having it be a functional component will help us manage the state for that component in the future using React's useState()/setState() hooks.